src/aws/osml/gdal/gdal_dem_tile_factory.py

```diff
@@ -24,15 +31,17 @@ def __init__(self, tile_directory: str) -> None:
         super().__init__()
         self.tile_directory = tile_directory
 
-    def get_tile(self, tile_path: str) -> Tuple[Optional[Any], Optional[GDALAffineSensorModel]]:
+    def get_tile(
+        self, tile_path: str
+    ) -> Tuple[Optional[Any], Optional[GDALAffineSensorModel], Optional[ElevationRegionSummary]]:
```

Comment:
Could this Optional[Any] but a Optional[List[Float/Int]] perhaps?

Reply:
It should either be a NumPy ndarray or an ArrayLike. This is a method from the abstract base class defined in digital_elevation_model.py. That file has a note to clean up some of the typing now that we've upgraded to NumPy > 1.20 (where typing was added). 

This change added the ElevationRegionSummary to this call but didn't change that portion of the signature. I created a ticket on our internal backlog to make sure this tech debt item doesn't fall through the cracks but it isn't part of this feature.